payment/src/main/java/org/killbill/billing/payment/invoice/InvoicePaymentControlPluginApi.java

```diff
@@ -260,7 +261,11 @@ public OnSuccessPaymentControlResult onSuccessCall(final PaymentControlContext p
     public OnFailurePaymentControlResult onFailureCall(final PaymentControlContext paymentControlContext, final Iterable<PluginProperty> pluginProperties) throws PaymentControlApiException {
         final InternalCallContext internalContext = internalCallContextFactory.createInternalCallContext(paymentControlContext.getAccountId(), paymentControlContext);
         final TransactionType transactionType = paymentControlContext.getTransactionType();
-
+        final PluginProperty ipcdRetriesProperty = StreamSupport.stream(pluginProperties   .spliterator(), false)
```

Comment:
Can we create a proper constant, similar to `PROP_IPCD_INVOICE_ID`, ... ?

Reply:
Addressed in 3fa9137